### PR TITLE
docs: update proxy_rewrite doc about the describe of regex_uri

### PR DESCRIPTION
**Pull request by Hazel6869**
### Description

Currently we only support one regex_uri, because the regex_uri variable is an array, and the existing description is easy to misunderstand. So I'll fix it.

Fixes # (https://github.com/apache/apisix/issues/7558)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [x] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/proxy-rewrite.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ description: 本文介绍了关于 Apache APISIX `proxy-rewrite` 插件的基本
...
 | method    | string        | 否    |         | ["GET", "POST", "PUT", "HEAD", "DELETE", "OPTIONS","MKCOL", "COPY", "MOVE", "PROPFIND", "PROPFIND","LOCK", "UNLOCK", "PATCH", "TRACE"] | 将路由的请求方法代理为该请求方法。 |
-| regex_uri | array[string] | 否    |         |                                                                                                                                        | 转发到上游的新 `uri` 地址。使用正则表达式匹配来自客户端的 `uri`，如果匹配成功，则使用模板替换转发到上游的 `uri`，如果没有匹配成功，则将客户端请求的 `uri` 转发至上游。当同时配置 `uri` 和 `regex_uri` 属性时，优先使用 `uri`。例如：["^/iresty/(.*)/(.*)/(.*)","/$1-$2-$3"] 第一个元素代表匹配来自客户端请求的 `uri` 正则表达式，第二个元素代表匹配成功后转发到上游的 `uri` 模板。 |
+| regex_uri | array[string] | 否    |         |                                                                                                                                        | 转发到上游的新 `uri` 地址。使用正则表达式匹配来自客户端的 `uri`，如果匹配成功，则使用模板替换转发到上游的 `uri`，如果没有匹配成功，则将客户端请求的 `uri` 转发至上游。当同时配置 `uri` 和 `regex_uri` 属性时，优先使用 `uri`。例如：["^/iresty/(.*)/(.*)/(.*)","/$1-$2-$3"] 第一个元素代表匹配来自客户端请求的 `uri` 正则表达式，第二个元素代表匹配成功后转发到上游的 `uri` 模板。但是目前我们只支持一个regex_uri，所以regex_uri数组的长度是2。 |
```
Comment (hf400159) [inline docs/zh/latest/plugins/proxy-rewrite.md:41]:
```suggestion
| regex_uri | array[string] | 否    |         |                                                                                                                                        | 转发到上游的新 `uri` 地址。使用正则表达式匹配来自客户端的 `uri`，如果匹配成功，则使用模板替换转发到上游的 `uri`，如果没有匹配成功，则将客户端请求的 `uri` 转发至上游。当同时配置 `uri` 和 `regex_uri` 属性时，优先使用 `uri`。例如：["^/iresty/(.*)/(.*)/(.*)","/$1-$2-$3"] 第一个元素代表匹配来自客户端请求的 `uri` 正则表达式，第二个元素代表匹配成功后转发到上游的 `uri` 模板。但是目前 APISIX 仅支持一个 `regex_uri`，所以 `regex_uri` 数组的长度是 `2`。 |
```

File: docs/en/latest/plugins/redirect.md [hunk 2]
```diff
@@ -32,14 +32,14 @@ The `redirect` Plugin can be used to configure redirects.
...
+| uri                 | string        | False    |         |              | URI to redirect to. Can contain Nginx variables. For example, `/test/index.html`, `$uri/index.html`, `${uri}/index.html`. If you refer to a variable name that doesn't exist, instead of throwing an error, it will treat it as an empty variable.                                                                                                                                                                                                   |
+| regex_uri           | array[string] | False    |         |              | Match the URL from client with a regular expression and redirect. If it doesn't match, the request will be forwarded to the Upstream. Only either of `uri` or `regex_uri` can be used at a time. For example, [" ^/iresty/(.*)/(.*)/(.*)", "/$1-$2-$3"], where the first element is the regular expression to match and the second element is the URI to redirect to. We only support one regex_uri currently, so the length of the regex_uri array is 2. |
```
Comment (hf400159) [inline docs/en/latest/plugins/redirect.md:39]:
```suggestion
| regex_uri           | array[string] | False    |         |              | Match the URL from client with a regular expression and redirect. If it doesn't match, the request will be forwarded to the Upstream. Only either of `uri` or `regex_uri` can be used at a time. For example, [" ^/iresty/(.*)/(.*)/(.*)", "/$1-$2-$3"], where the first element is the regular expression to match and the second element is the URI to redirect to. APISIX only support one `regex_uri` currently, so the length of the `regex_uri` array is `2`. |
```

File: docs/zh/latest/plugins/proxy-rewrite.md (hunk 1 above)
Comment (Hazel6869) [reply docs/zh/latest/plugins/proxy-rewrite.md]: Thanks, I have changed.

File: docs/en/latest/plugins/redirect.md (hunk 2 above)
Comment (Hazel6869) [reply docs/en/latest/plugins/redirect.md]: > 

Thanks, I have changed.

**Review by starsz — APPROVED** [review]

**Review by hf400159 — APPROVED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Pull request merged**